Tests/conftest.py

```
@@ -0,0 +1,53 @@
+    dpg.create_context()
+    dpg.create_viewport(title='Test Session')
+    dpg.setup_dearpygui()
+    yield
+
+
+@pytest.fixture(scope="session")
+def setup_logger():
+    """
+    Настройка логгера один раз на сессию.
+    """
+    with open("Tests/logger_config.json") as f: config = json.load(f)
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)